roaring/roaring.go

```diff
@@ -101,6 +102,14 @@ func (b *Bitmap) Add(a ...uint64) (changed bool, err error) {
 
 	return changed, nil
 }
+func (b *Bitmap) String() string {
```

Comment:
I'm not sure what best practice is, but looking at the existing functions on `(b *Bitmap)`:
```
Clone
Add
add
Contains
Remove
remove
Max
Count
CountRange
Slice
SliceRange
ForEach
ForEachRange
OffsetRange
container
```
I agree with you that it makes sense to put public functions grouped together, but I often see related, unexported functions directly after their related public function (like Add/add, Remove/remove). I woudn't be surprised to see a `count` function directly after `Count`. The reason why I thought it was "weird" to put `String` in between `Add` and `add` is because it seems to ignore this structure.